docs/examples/multi_annotations.tsv

```diff
@@ -0,0 +1,2 @@
+Term	Term ID	Annotations
+ACME Communications INC	acme:s-1	"acme:p-1 " Value ZZ"|acme:p-2 "1234"^^xsd:integer|acme:p-3 'FOO	BAR,BAZ,BAT'"
```

Comment:
Just noting here that I didnt know `"acme:p-1 " Value ZZ"..."` would work without escaping the inner `"` symbols!